cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,68 @@
+import PageObject from '../PageObject';
+
+class settingsPageObject extends PageObject {
```

Comment:
The class name 'settingsPageObject' should be in PascalCase. Rename it to 'SettingsPageObject'.